pwiz_tools/Skyline/Executables/SkylineBatch/SkylineBatch/packages.config

```diff
@@ -0,0 +1,8 @@
+﻿<?xml version="1.0" encoding="utf-8"?>

```

Comment:
Does this file do anything? My Visual Studio seems to be ignoring it, but I wonder whether it would be able to fix my System.Collections.Immutable dll missing problem.